pkg/server/ca/manager.go

```diff
@@ -99,8 +104,9 @@ func NewManager(c ManagerConfig) *Manager {
 	}
 
 	m := &Manager{
-		c:               c,
-		bundleUpdatedCh: make(chan struct{}, 1),
+		c:                 c,
+		bundleUpdatedCh:   make(chan struct{}, 1),
+		failedRotationNum: 0,
```

Comment:
no need to initialize with a zero here. the zero-value for integers is 0 anyway.